### PR TITLE
feat(FOROME-1417): Adding a rubric select for solution create dialog

### PR DESCRIPTION
**Pull request by AndreyUstyumenko**
https://quantori.atlassian.net/browse/FOROME-1417

### CODE REVIEW

File: src/components/solution-control/solution-create-dialog/solution-create-dialog.tsx [hunk 1]
```diff
@@ -62,6 +87,19 @@ export const SolutionCreateDialog = ({
         })}
         onChange={event => setSolutionName(event.target.value)}
       />
+
+      <div className="flex flex-col mt-[16px]">
+        <label>Assign Solution Pack</label>
```
Comment (Lapk1n) [inline src/components/solution-control/solution-create-dialog/solution-create-dialog.tsx]: Font-size: 12px (figma). Should be in the dictionary I suppose.

**Comment by Lapk1n** [comment]
Let's invalidate dirinfo data after the preset is created. If a user opens the main page after preset creation, he won't see it as there was no update.

**Comment by Lapk1n** [comment]
If a user creates a preset in some rubrics and then goes to the main page, he won't see it in the relevant presets until the page is reloaded

File: src/components/solution-control/solution-create-dialog/solution-create-dialog.tsx (hunk 1 above)
Comment (Lapk1n) [reply src/components/solution-control/solution-create-dialog/solution-create-dialog.tsx]: fixed

**Review by Lapk1n — APPROVED** [review]

**Review by EvgeniySobolev — APPROVED** [review]

**Pull request merged**